src/ef_kasamizu.c

```diff
@@ -18,17 +20,36 @@ eEC_PROFILE_c iam_ef_kasamizu = {
 };
 
 static void eKasamizu_init(xyz_t pos, int prio, s16 angle, GAME* game, u16 item_name, s16 arg0, s16 arg1) {
-    // TODO
+    eEC_CLIP->make_effect_proc(eEC_EFFECT_KASAMIZU, pos, NULL, game, &angle, item_name, prio, arg0, arg1);
 }
 
+static xyz_t base_offset[] = {
```

Comment:
Move this inside ct, the values for base offset are 0, 45.0f, -20.0f, and it's not an array